lint/rule_enabled_test.go

```diff
@@ -115,35 +145,62 @@ func TestRuleIsEnabledParent(t *testing.T) {
 	// Create a rule that we can check enabled status on.
 	rule := &FieldRule{
 		Name: RuleName("test"),
-		LintField: func(f *desc.FieldDescriptor) []Problem {
+		LintField: func(f protoreflect.FieldDescriptor) []Problem {
 			return nil
 		},
 	}
 
 	// Build a proto with two messages, one of which disables the rule.
-	f, err := builder.NewFile("test.proto").AddMessage(
-		builder.NewMessage("Foo").SetComments(builder.Comments{
-			LeadingComment: "api-linter: test=disabled",
-		}).AddField(builder.NewField("foo", builder.FieldTypeBool())),
-	).AddMessage(
-		builder.NewMessage("Bar").AddField(builder.NewField("bar", builder.FieldTypeBool())),
-	).Build()
+	f, err := protodesc.NewFile(&descriptorpb.FileDescriptorProto{
+		Name: proto.String("test.proto"),
+		MessageType: []*descriptorpb.DescriptorProto{
+			{
+				Name: proto.String("Foo"),
+				Field: []*descriptorpb.FieldDescriptorProto{
+					{
+						Name:   proto.String("foo"),
+						Number: proto.Int32(1),
+						Type:   descriptorpb.FieldDescriptorProto_TYPE_BOOL.Enum(),
+					},
+				},
+			},
+			{
+				Name: proto.String("Bar"),
+				Field: []*descriptorpb.FieldDescriptorProto{
+					{
+						Name:   proto.String("bar"),
+						Number: proto.Int32(1),
+						Type:   descriptorpb.FieldDescriptorProto_TYPE_BOOL.Enum(),
+					},
+				},
+			},
+		},
+		SourceCodeInfo: &descriptorpb.SourceCodeInfo{
+			Location: []*descriptorpb.SourceCodeInfo_Location{
+				{
+					Path:            []int32{4, 0}, // message_type 0
+					Span:            []int32{1, 1, 1, 1},
+					LeadingComments: proto.String(" api-linter: test=disabled"),
```

Comment:
Is this leading white space necessary?

```suggestion
					LeadingComments: proto.String("api-linter: test=disabled"),
```